promscale/quick-start.md

```diff
@@ -0,0 +1,61 @@
+# Quick Start with Promscale
+Use Docker Compose to learn how Promscale is a single data store for your
+metrics and traces with full Prometheus and OpenTelemetry support, seamless
+integration with Grafana and Jaeger.  
```

Comment:
I agree with you about the salesy bit. There is no background information because my understanding was that a quick start is for domain experts and proving information about why docker-compose was or what docker-compose was not necessary. May be we can make this better.

Reply:
That may or may not be true, but we also can't guarantee that only those people will be reading it! Especially since it's the first child page. Let's give some orientation here, so that people can easily self-identify if the page is for them or not.